x/gamm/README.md

```diff
@@ -86,8 +86,15 @@ All tokens are swapped using a multi-hop mechanism. That is, all swaps
 are routed via the most cost-efficient way, swapping in and out from
 multiple pools in the process.
 
+When a trade consists of just two OSMO-included routes during a single transaction,
+the swap fees on each hop would be automatically halved. 
+Example: for converting `ATOM -> OSMO -> LUNA` using two pools with swap fees `0.3% + 0.2%`,
+instead `0.15% + 0.1%` fees will be aplied. 
+Note: For pools with low liqudity(or for swaps with high slippage) final discount will be lower than 50%.
```

Comment:
asking to learn: why is this so?

Reply:
ops, seems i just invent it. I try to debug and see real values -> it is always around 50%

Reply:
hm, i found where a got 25% - in test of TestBalancerPoolSimpleMultihopSwapExactAmountOut

So for out i always got around 25%. Can't understand now, fill try to figure it out.

Reply:
heh, more exact test gives you sometimes nice catches
found a bug in out fee reduction calculation, now it is the same as input - around 50%